packages/flutter/test/widgets/draggable_test.dart

```diff
@@ -3104,6 +3104,92 @@ void main() {
       );
     });
 
+  testWidgets('Drag feedback is put on root overlay with [rootOverlay] flag', (WidgetTester tester) async {
```

Comment:
Nit: I think this isn't indented far enough, or the contents of the test are indented too far.

Reply:
I think it should be fixed now. @justinmc